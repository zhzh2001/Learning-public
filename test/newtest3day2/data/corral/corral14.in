123 988
226290235388532865 325643857853950083
471005765435885660 688755534437039701
672636876819878418 716905887399133705
583870892660221727 317766611227711965
700999635012210627 751652857410849003
865431602028159034 371736785058798533
694310491753321308 382601877428271297
175035311281683694 641245065410168992
572652054306025959 680431251043460236
741806027983791786 595931061903530412
653945467830239297 322546963823736002
903131198031930999 310432587219424222
23588889432500854 703211187961471006
254806797303558875 863249562452618856
736374456844194283 560094341381330351
745262904851711437 4453218474000358
211007331143678793 16935872244176489
92466882207439585 55384453332059416
69185935257751401 880532162073868168
161989109280936197 429612794695964262
553368732718807925 117867334828403392
568112153450705468 50901856787972846
82361616394259094 649315444015622025
709468845780462262 117144364552764274
107334419326157702 558156200568176348
833329349888901580 765601979226730064
239603024208449173 771033264725081123
658704229832193187 346229268604057268
451095096732430818 751532843086534458
296105899535966373 717930939147837594
24191183899392587 340228782189450504
673755298815239660 398546147988910188
908800115371970093 260334335230883760
472702462281493774 545203407472025990
951008101517693823 857495650805410763
812968178067863171 702728280697192116
358329053892617158 262028622383007730
368624578093354740 687695198507000747
991921985710287272 818163161137049586
813770631712616546 975328186827854508
282312716233207379 306148579900930414
322505590978967929 112730360508370068
568049737669047229 225262666562604874
397591073991727409 239153740971248246
868731344126921466 374061595741187794
318914926735526347 788088908606149938
917395609532669789 672131257873890153
974823085471958173 583227682803415785
223869586778923481 242644806705660287
527984735596901107 745919604266251592
900071192497569293 845410993752074889
390426177379795101 256249967352096766
669517760778442486 580560257544891651
515058704700869445 907011270307860538
537004464641051469 452947324032920195
689355951834710769 78941396073008437
289297534445331066 768561963159601607
228235399104449155 202225899720568328
402382723367737413 168030884780966836
789793412256671602 49590352209961479
982748169450114748 141535441317669545
187309365504738909 403933691755253067
741526288791990559 205465902043264269
454148345675751673 552314191220827865
547087431126454745 354574107411716105
24375699399470200 971278959235820315
910085970246215820 293180239772883315
112028471827609171 910499907019697866
165209999428073972 8349877257460500
895803627792168635 422860943571236612
535699639337815215 853502129653268185
77120780426549284 115861101923993761
943766429757389955 799202611740561413
762852532045438168 233822455597571779
742965150600911528 622278874067055922
215197201494826097 657811648932934384
670531219368013022 418770783408501892
744123053338034797 732047843914392839
85623342467078985 24192025788622476
623672005395123647 852309610662514235
679711940408872496 538878669117470665
357193053163220866 623755552168128355
73595832944141305 782185995441464595
47552799735412092 444039967879037683
266516744899706757 164324692709947950
453309035750586498 219218816406853055
538489177200915645 528517980109005257
48493544433548853 910843823087834577
190146856694252945 192348017035527229
389237517764124961 971181882099271795
451927383754785435 206212041455611439
641455063094841138 27913852946380574
268422395658951424 638579411985043650
606909238044654823 307672079502779743
7913774312448124 470797619048506503
203764430912237094 374529190070188701
159703399201286399 508945291295205406
284915676826521125 953093469867249555
427733205993934686 323155102119106607
899179663960101274 315631511275168833
956680953401357465 810474577174702982
378550021080160938 925354019011031940
777284709315340760 624263967389219080
718394835177847337 893059869806824134
41547609908863169 996217128732722672
223014963918432052 427658728859712497
275337704313032871 626910467833362824
560243615653559228 373165793326785721
320713247349368662 110965891575290825
531457790913965743 591175246514243166
628200233627223506 462295482248114155
988105366543973103 499739844052105172
493974478844702906 818488695239861144
249496956482127999 271430281373946782
954277574260478823 57331354518836443
920292194047989024 105148445207137789
48960337679032928 707269004112748469
986393135552335545 319598858691244256
843207281970555740 672698895281245065
363452098381379702 167712216800419734
166324360166497430 138929048736941259
85217970628685804 92006774679567729
756795278238973065 172702582114997840
556982626233954270 807578840991836909
491704876340721191 106460598383328532
738531819613768865 638562430304898186
801218298718129588 461888720863021477
123343779526080223 47258707620072546
726215877103433835 314655222527198843
417126771221143574 200879091632420951
174680326087535751 288799075853705434
235405265440672579 940957816229332311
596130937587774068 72623620486315230
307378964580400618 110337565118561679
85593893523564142 946644035253108285
985059320908404668 462998940880757765
616844946926133879 142226567990432954
743739362296472320 840743647610857511
251772127911822333 835342627173178695
350589911471886916 4512829389800946
779349436343844462 208942011356735507
108751845173636541 589618225973509288
255432452050484172 114185849630776051
650570030864452224 996368587192137688
804350118267375199 954977025328102899
715211726959936805 50157186386219370
54833866330360516 134687262595716957
818743293727619637 330135902028213277
61383468588202672 55382355051942477
966413887235103906 274949150931401945
26015246567003390 407969406165060746
665031341369535162 759525879556959211
594728236212100921 832346592338427747
443861098621034969 922360716020985011
265314007262926654 83614506578376052
429128618154593130 259660641250330844
116521772392807686 571163679957192321
986823334534017151 375632372151377321
151362651717894754 76569912079391838
754796293743671996 771056081642950268
715374353338540904 648869150344670612
675548342587211552 927370339553090652
26570027981211167 363676489795496308
256961670915755117 117619325413093893
769972241538633381 301872871205321325
922950420807030297 919057647170919538
750291348475098609 604427668436294739
16974212569256990 285227674123193118
10761718244703612 929964174668075170
625736994439898784 420432266945913947
942410345853059541 731735165804438790
571417468770871875 553776286596807057
807413366793647475 541773544082064549
145930758426642938 883033458093186385
65253498327597919 994348942046307853
562336094961057615 727333652648147618
810216192332724686 756453333738122765
521526392419635526 244111725763719872
780533897251362640 272630998046875684
865311121657945808 676786495413414167
875381320205044422 769292858362664050
431453888440043965 885459076426748989
5044740194607495 882470425304644052
618598353176232080 893821536922025746
399766522347141863 298895119542910763
1864408922901936 187091848803641949
793549071766247015 357132398476481228
760870449766098741 822449835869945513
827459888937348883 520609222769350141
105932677962127677 128679867105021512
715601682774762046 614512574273881747
786162182694815885 763197266378457280
854187695990784540 707138150463784531
445644776374932144 239527228400763035
673175243540790535 101467887805809537
932404633447986886 261795586569859791
936365213534444725 632869705968258771
413640189835888585 360297367416537097
362778045738668064 898997783659274829
178914560997377373 827678558940801582
782012080873919345 823995748626535289
829710085034205235 364520125141692245
810812920021725343 822202477200733475
42835157152565219 293971231266783867
929012936421483900 402022472070150968
774439621793232714 790781386513874334
475374822901384802 304091174302544094
458329245181197767 360595840792405843
831492738346398742 751772959789764665
965139745789502245 983141546513874338
310094079001245380 415773642520041911
252891823479484465 167612358422409322
125348734562883775 487646291321222659
895076521091050995 599058487306234714
746691000357184916 339788965911516771
445528864113220170 515228315324151233
372477501587547266 221409388701896710
80953528645347107 203624619303584055
103087197415178041 966302008310290872
278281715974015133 296212780816787343
821590788795390284 243035971662436248
409860192853608813 343908441861809074
366063211555705341 108844489505559154
156795867977577812 140173159218537706
673320914011418201 854034893085736544
216592504124260936 385677281804787102
971956869105987304 130547289218230826
841597046295661111 310080445549218528
501995340614986351 983788907829702828
750826646166060098 777213863902420730
662285676247013067 815546766825500065
141378151915980850 335352421999599861
58622237104596725 669575294890893434
202952362119004864 867541776044042807
311847023500750089 957747966458253447
571442828537892798 84269457784057987
980678940891233237 756288407197547752
361913889341365249 512119757261809606
914366369467899770 861268180315941680
134432988407144694 347552188083936856
559400994010924664 737456645385105799
811623163289881886 496098638531548980
117231124250398416 564412995021122144
574898953719053490 311893089019269754
288542589558101271 532416745763096174
445977359153012916 518217415310644469
914260804712576394 647317733092670117
533863788072923067 252813510594381142
238570169950330007 111720273771362173
930269447588519367 113845410651216939
147295499615871981 206465364223488854
527613460274707843 428592225148547612
127299381297588297 965402591355855667
731434355756142992 644798106652364809
8007314225615418 716606383138482830
101764832035749981 279679348832508939
831929917277211493 644356716034949
525336416070856678 235672651195729763
924971023816778254 64047253534572914
121518095362634812 408483839176107576
870022789404129821 548892693031729774
512932687273410566 873354845447164916
875708372715497173 991052642673391882
519306749172084966 332851080879166856
209750248694536605 714871861409919790
192009495931288007 734407537162616926
942027034950374816 117594025781069681
788302553045909625 995689940669045393
56970594221816717 724504795118943369
200685779248360716 117924559675550033
509179418128228727 455816456943938675
466285983767063979 902727600174468527
625538162526787262 688440064347191840
669784948947143968 31916602665818129
102679954950353030 742366728395067443
334771505507023360 530210594221101370
283358231045891249 377832523084733318
802610399161318642 152885910779998594
903941348949795068 807928493504567721
22867471927040984 136538585724061650
41978040839583344 997149553330966059
588368443477861995 505312300638527631
698863594377694193 763423423183039029
336981512128845056 916982793846778699
964947316100674582 658866866949277478
795884485528845369 628451413013531297
590951574571782839 375754050925205887
358238421933790555 350841556904189930
206106431599366358 851167486906808984
320842657759666582 193821789330177056
414250639031613753 267779282447743145
538240570731475206 89257503607767693
979136024143874895 547891252841285130
238271013613116308 69743459255384742
572119303759666251 686009996027444676
806662124501100200 748084196115651083
255125012124122864 209838595525225946
547466047208097316 953103980597069633
917908775414783815 783329274647491940
884876205548251646 135408566274310137
607139136101810607 307839344863313726
630249557517881752 39908225964954543
625653731160270707 25662872947614314
450670301305594976 468922738054110441
323062316040761168 152097167903779164
514684257260001158 618807257693400385
239619580648698639 374231038845829455
620218365075843023 543349015313634127
522417783505937638 476740322220354723
226210764905768613 212254764081080008
998453949098711501 449116049120214339
686843026145996144 811834531192356043
52636129385745581 32179708101104152
128524590076380556 421599578654743075
675192128922022320 33960679782128444
709669524695754668 831143049322922166
981530428143650595 946509564916531027
728158974754480678 249152018119722996
65622501407859114 421283891649642025
865507640523315662 236156359543332401
954756295702907485 731302469074768465
758485746822015908 650064959576174922
163863139795540263 266742472168845964
134408300863036803 391274988626910337
478993394056256003 374184784331709279
556014004691765567 277566724317427965
290533926176711622 541776007467273859
136693300061640700 694718064960362739
589264701199489474 137293801218559425
112511123518308920 641877429499488434
38273238315312226 538355752783195522
419917108020084106 681036779911652162
538779092085160154 269557337583768097
86063755799385012 230796215837290112
426050705291753226 101366693572776523
229071373687797453 573970970516287743
394356586903212004 243356400737612325
337239626363410559 410584402338721141
161761272993055046 249987668403643709
685520637142124054 423325367929384039
340079144959854257 490218471411497233
394470396589212225 743340160733179616
836856146521215444 418540760785661330
175772731126748419 250002113677571043
844824413835816566 847433097987567491
757517516157233420 499116394968988501
598028141534439646 204442776760587190
409205473596965447 41629979755068108
878536156406143339 385637816097003539
320243041445415804 635809082942664610
16396476785869348 539974336477587932
978015398632092131 701110916883037534
401306945641913217 829385190939107488
287437206108360942 6981824350858609
495996228713631599 88746390904511005
861044259242932402 412228257745581627
705535524007302937 837852123022058968
358621750018033933 937469057099776560
227178233145553306 962703176452196830
263445167683075540 550606171823450920
403193181403566956 318626652676496642
489943356167888779 263877202928264754
542229437868288687 134983521716925344
854770483109841142 435422264592838620
952856325548372313 8795238836904176
611285000556663670 453423848401812712
927587138930111521 987870238893497060
423042630581109086 168423971908319605
955406352353032125 50111737250921215
721242968105461981 881018535688437549
602416077078395487 673991661379401015
255428487439240592 252992437839703837
224746859038064677 926630849347803020
844363322671905828 808332415661071610
234431442225443817 648024058873603465
475604190482388276 290354489932197001
29267774140099065 605901176566658505
78789881773708816 453246113117219980
152191459503753687 435871444741604778
577076128806655406 372566834201807407
93830302575286595 749182614649756022
733370520706753665 747678057556810105
125513450631042706 67587198243924379
287137461307771495 990492010987675043
928348561210262454 994912161682773967
575954807447660393 246195779735116844
389890340622070924 727086981235247826
505636449489993151 304278846887463078
23887888989264991 96466897934110225
471376536054240458 325429880524345518
813428609895017799 257777697877152466
517178547557536490 694755367957968817
87468510353963768 715136738678556334
314618226647509968 796319280538487820
402969472950264323 322215478508512531
499185887345665173 431217094192570255
18290293846428258 424348525427212068
502558848040037743 420610341563478290
363746190496815790 81607786151252169
221223558823756879 399205162599013554
357447597958916297 635227300936221107
300425446609277846 393728825285173223
885783306889881978 260819201176436951
122127573808478020 587270980659901906
439535383420299749 557486533481588724
377344250923886347 639146742274626171
336140602184516031 126525249066404411
954497833544615180 925153131770951912
930643149785562343 508081188589627757
953391773576849714 531395020765647055
696181713106955617 283586960767340351
72109775938595878 131786279638502953
808353467188668695 767612344261119944
238297928298089751 65962671927826976
329392011233354146 756529155317977934
478554282216289710 794561173024220733
435189179383174162 747722258891638505
467211411234521026 832714985478545893
433617994914357664 97053291002541309
583613250914488553 528466921241991800
35724027627954627 114744322901572323
829154137431484797 667630098959488298
675504824213245396 85359979310488820
14902780494195312 72097759687050174
764308903862931910 914184523529997547
909563919425662867 439269765203191355
489105504515271545 944959760569658220
264482104674899791 70723802817165784
443155732851648202 492191316533317472
119218416793581365 350429056926536796
7683216918294876 649321891341503381
323097709753464651 935135322820429337
32566972876745349 925752335731222259
39070206793504098 774488470700746188
516867722892600431 640998885815385182
435039284432322044 678776556312143440
618752077434790329 660294157076025478
596678251529895094 257338281067138784
691523030693522246 845504340374095129
974842236305552666 903193197164303617
866595460272613239 950008940012163167
245776529609549590 257277982894639784
11771523636011606 980449090082627617
626565482222166302 210755863968882265
767655731626853841 316053755262194248
294931921438663867 100170169164824582
241210493676150000 762542484839526640
424507015380579000 138707193728994512
697241441244190356 233016580704077139
370186102312336240 492396378856670995
775360571354707711 794690407324688897
317156883651271535 467167841316168537
932099839459260396 207389234625220109
185927848812475294 351320570430557330
981058983030127986 981410091667971363
648844288838699320 352388781890923840
466701619352694642 529844254259792292
188671845190129908 144816185068271481
74746955465490068 448086034090587420
484656179778989593 121401291815031617
408754253155305789 275247175442978933
934524639528230224 170552499969731041
247313263195553992 98219698539467107
496188813035232093 665238218479918618
875077563544468147 476100832139693412
351184491484061031 877466735803633026
37410647574345581 99160112495242636
230788340306431577 695651333595609287
673354336157173169 796641049764281313
141523657015548577 46504327571919282
56804368336820054 820233360445571703
560465251598576057 310884864318035040
186415182438542985 835580540360228735
304211384065392798 874061637274093486
367762515680815404 925146312897478975
173165911158206050 216017405687801192
74714521251368796 568813969110554086
336091409648629215 872177093881751258
311057120879144730 569142456269159019
480585565622367686 170039488720582929
37549217825703867 543402245481869885
484754809685958716 774334746442284480
825250125002756323 144795275287338576
58728104682001135 149200261236998700
4334516380264385 782348273896486705
627824248449864810 58834736832562779
218394495117586550 774036118433546362
660167474215367741 865430487383839622
318896967862565275 303335799880161382
584397930386334823 311777157428515481
153274749802731152 76001039758061989
250404218899609623 289954472251795168
988888014313536251 360465925678317727
412158174758093831 842947806108991688
536621890391340321 896177302069510663
420467568308759394 341459613585777090
365966946240767677 704500406097910351
758682791868184612 591974362669165037
765658969976823510 659890733180378136
434683721513037189 712582126584222785
100546549514798139 360991984063890214
389849360801951357 978492502873943103
86419215630090285 777722818190563768
975860404750192093 631465610750532667
189604746572559084 727405019946416496
174376631709343292 501280559430622181
119224387334890765 997027185152665809
834790345654491554 705523028824232610
122972630916572675 470678496010083179
431612670477060010 34374425412333249
170585526942822876 375562933466811983
137597035993835123 505671237557955316
226617590721452817 747382940999380724
693489767822818907 641475955694408474
463589300708109160 982418026432944069
603650649727344379 459524676421645868
105076251053279590 644442883066051565
993596056781303482 875444876403046290
73420515943546552 912998713881396388
177781526209593482 963757443047700263
662580015345321375 959979430518525927
791034964855186077 713613423776809243
270977450174884437 216846539920619627
943945874592418413 788485481412932039
161506912897606666 626122311272890684
4313810628717372 169572147817194808
110578092028095587 924540940809688339
179591081320458697 637120632621790765
237411506935885672 24103835738527508
986619006716057017 631826553599929630
479078094131906075 834703680454585261
242375103607266621 249806084482254275
275992653371038076 539744352513169864
383865563983358235 513719252405474047
332687034886389039 105989574214445003
387711080140267637 586043057217867740
212073659091890989 476687635414331655
324792419998194057 272772055307935581
105138684016110314 740574745575351900
888640977650095907 181434372137833190
914772207352191632 670319458440697275
581205477627213415 68874829247163712
184594536724850966 274007426892917658
20434953107951807 932624355812844201
896134567597599228 569816092262343683
297330476897365574 446848931478354047
880171887151982506 207697054690266086
942426019597931623 83009634899263506
225106294218638535 326879830788223477
181244087978010526 659959465279251033
15279781522695568 436850579161531643
173742463571715462 639744397759877646
132114440350196655 709748769672558811
317845819693625531 807436752845898341
851427551231603973 290246949314523379
398406682156403763 753834381543539543
794967459772852623 148620128646380378
737713108056072849 420582219884327400
658215268267212427 169391951295117766
581710606902665885 916875090005577055
158187644121725449 256353274899176912
334197727891752566 426571824020557088
299580849103713786 164680165426665333
949214630357811306 200706665404801320
334021167386234602 666823697884953454
318126385741016079 562853092298168336
9998919303672207 861377570083328102
845336299702406429 892351323327958883
398082960692655783 49298813682302174
687726118609365265 644899264374748600
848527635669821320 527243868730829545
384044074579347039 86527256656441997
697093986054200060 806840536304007695
827095997327656817 429457051621279813
88393379425277932 23544653734506878
724139142414429245 158557289357473727
783067569497482148 685701138634489839
440416134916033170 688606530771982263
683050372904312044 371478005044446730
693150250196576965 40990231953344009
496306421961560767 46588288846532880
21783457860860696 143223989877715768
577005742996459054 700345635227561436
972427841730725855 489380581100698859
291292328823428447 835250229824031761
817877885234783232 407513864597530703
806157738321651497 395623739808589433
644902369915638076 183191614138426358
897222769633451286 952974688309281488
287048940515460107 915341904381337530
956303565790852725 3527715837954303
316506133301611376 441733111774256035
342379506490059573 636175287600146615
378206725690357843 753245537976104341
875268010927625749 361747697819292808
160678083634932162 198994346358562645
161436247889580227 267464499614593696
646798042568602080 484865215304583241
489680108986083389 14977736560376561
43795918197873851 869998857842037806
198961355896168303 174230772243018573
454073554980678633 799664587747381453
858470809875662127 795533585215644802
952474376272686137 37129497358981015
738046242787121502 120955646742404393
609746370572203220 422774115147947110
766959295897310414 185291333264036762
470225095676639113 72375984766805022
817780505275719240 954280611075326772
25576796211180312 907189448013618296
899165231572234320 203342463975558120
992209185931048519 84636059761590229
960774193332812630 358361423676726155
494441461085932304 927991894385283192
369686941430323356 659629833409093823
322741103063289016 619512135940121171
330257894362892694 439636345704149381
398678661791986416 204776890832031639
984712544134767055 181013698102545676
100682269799232257 595200463080049792
771287257568001756 487714194416926047
686553626704022023 717478752251794249
484575229547151455 617484458483610761
577953633015076525 719623641314776179
392087510579980868 968753607089642452
696501940300615616 439088078193270610
758345432657152862 326283365115783880
687153986114517922 108288458143108350
620981415295003567 774280874118754724
410409882126386302 464209208272153248
115406778089459253 615269774221726313
124549160952818414 738688344442686493
685236020576468997 207012643803023554
692827033436715723 115143440705086390
896081472733142273 242319463745981158
56343446839564070 256717314698515077
867983572480211225 666741643747276096
393798568939834221 90695068219943742
890511516042433534 155779774188707836
8857766297511020 667884394624731319
260368000063916134 524302608243834954
979555381653230377 589950136528918383
297163857987623863 457621162600602738
484647322760251615 668789773529817275
136198904874291686 571436742021991707
69424512076814469 753497829859006544
164905166781311155 107006548550967362
280771083796630118 752245652703563946
795381209698121263 958529110854298679
32205613377966224 14043664693888181
578085798894377681 757938447913487180
610400767677047461 114391350080520390
132698728695239256 52605872915683716
886761064648928371 787588235759912218
956370017057134904 830061253806852239
341556353537079000 362521052591693517
417320491663685967 597088513628395568
663400722094409936 567356639429287035
446886414880564090 546403245745268863
927732925375379149 447591641051011678
949589292566304322 4700469759472438
31299774870253132 775032790781698677
931236707503612978 958232155885540147
327932987034905770 854690806450303894
582789427886416491 213925388130992839
260155498188225079 485304715874037664
718967122305370969 742890213864013932
968170302380349108 707221392266886662
867927810201014256 493310387127002903
532002428851170814 439776586848051878
503978226125615882 121103847176413083
953164976764457612 971798501570290524
14917225767929562 992180082763390664
535349888032003094 675621827494881023
543488201948117382 329309596285744794
484585177585861357 748167723559661763
754882119201735505 145493893056584496
426267841853875892 238516778706387770
812656343995008825 788557147238169845
803077660600426399 316355819553652095
856544421149767767 342161171524321567
738341166053500370 859494143483030287
929969357011928118 122065059233641931
476722247372801960 373210836551181906
479183102638715027 450542018425773389
280312806089139138 21355881192707775
571904063595951362 640532189215257415
795153113541404053 697423637253447840
569901314223246062 757427638033035589
75598103385080628 69766735775929294
90005133507958567 463401144895925926
806728446906826850 348792705502315742
234759864953650308 205817113769445873
203593177316160314 483220704856368376
344176244266344961 725133697153374051
103410873777618149 157293031520231796
756106812537919481 591621346894530233
447078455840096707 203162108525451329
460902067426911153 861733317702956550
242287096109790245 706252261725168137
492569578248203051 381693193248794413
566549688549605853 790474932371132096
708890034011177703 199032654468733550
726838907389491614 178315662007250957
589210433703586210 241178340807390733
937237127340241113 108355334649479611
35516119632837312 793632972906056985
12509456775863072 719039612838617908
75824994694862683 104289374508090259
817535678716937434 455278483247626794
606357162702885491 511507931405397049
718948242078947914 61241081915150363
965820338107856733 330042151642871256
497917845302121524 244419146360942339
739258821078775006 945061317760155145
746794795426979314 442035755938430982
713811922800365511 813450043708976282
68121651898027519 158188713664833531
895882975857407547 187645826986849248
704892863955836417 243493950842716945
526082465280781188 93876245676377463
641883440846655704 895780666980055122
805878795918061913 38677534167711948
101611152878596145 50896457528432272
645597396769198352 71634145003078228
168985765487292186 852254554969009706
492140704383063855 967678593936830678
500396796892258113 441343832477697285
136526002486052309 150916931476244084
197597553241718540 866012359592705786
363971118830114896 208268557915734997
524307953811264195 596896262196736495
328797089740531690 455411671421147971
727507150566407248 50858920434352249
497199896294290979 193788487454512052
918284434145424363 674164534323973569
11878430689291232 711467069998538807
889936376800164191 562587106828339112
907541044173982841 612023294506109234
463160953023768814 798021466845914356
884098969924048098 158924810540694680
371844413731218085 262131747377517555
37841106424814975 488229218039459794
397658133050572449 606001904753841109
30566865146321407 327609284900200090
122641600908767693 92655961520664990
624041534656280234 857855577804148874
478219867469958248 365165151932866954
322150878539571612 747593735472264513
659786184276341133 28823031091601900
939319658608643431 80281037364004346
882592572271951487 898411628983057316
833257761380026054 406119622920786651
712953724058022090 636392196508728985
845520508084711652 787190894084764236
632510747572160328 517454304956390672
282027704495227085 961729546528066851
290192941687214820 93592874763177535
644913533539135803 533158065789054308
149331821471606808 432907154717034398
928388673369079643 464069693042888949
925303617332904780 464280545502851364
711023368573014337 690461962406007987
872855014490253626 241275980635242513
373914988883361035 173916219211279234
980181431572673994 514057195784624628
471804314604054270 908173494170130129
157771767996301026 520242633677540383
639523485581918828 913520294360793741
719533039423520338 402365799674815388
475870500251355459 862287875758832678
163494113090690225 843813796622511548
901641640313315672 268923298306840609
890746392414490812 4705819622496493
923105266152745065 42273286257182087
16998902261282966 967724719590202241
953268127530943385 440937277269522416
283417094570334609 819270415213221745
478034082692884934 769735679240343053
477040737095890847 142402166336727607
62400015536766136 875363333412334130
646157991944362510 968576859736152436
58968728237084435 383907772274470820
705241663841146773 910120644487504378
576672311886214745 310776337916452723
982398695194407714 582328912299965935
549351694506085680 417882920955067170
319251450500358398 784439737352795260
922664294424682321 880657182632640343
549794023566131409 123938683837440568
181880856952254927 599027921570916813
865061750615444677 996949731338544111
400899194177654925 956175276468194032
183553370957690158 223001603221398181
292156573275815088 692417091342192831
319879658835012207 925911811468208261
966002521230909025 697912381903223683
633738366043905751 8705422995234973
987547436635486062 578145895185264061
316818656778792594 446647969069685010
186734870565392024 659563124421652315
660583623095543791 435928993887696295
442554445645046297 512342632416154832
851849410784494718 916041931026566600
592236639101185009 607322876290636961
129741407881705996 180507976068170913
254661642275346409 605831097874427627
804416599601083327 555155247553617159
849010010310319441 596797640880329480
950328243704780995 8310143089892604
409701934849376992 499139499675539466
167465268337414855 395992345568703849
257480837406652763 600809342116281133
197664852840164257 949711559803605179
535082216635940820 418063070228159610
287120756867049 39270269325671535
972915969997135375 700389256690107174
80784530109836827 469443659196751685
19348671095192910 833203242605559213
949860975822643048 974254372078561532
803741629900794146 646380828440574867
863051443568181694 915713901323105157
839834366377297958 651785731878071148
147267203974568065 99104079608922499
517076438414264767 29980890875045756
982572805200631936 976905103445718563
415700056671662672 382469359329986421
388879843678517974 541685302487534978
923775238209573161 764233174633320944
235070133370361568 550560989000067285
119704236307457864 878994088337729059
337308525981213209 190867226133372033
987790343171029894 499303326605435635
919004223711874622 604901177467299691
65949188813219494 900171409537690157
480405304669843932 359978448157860856
73478964966236210 547382747417868287
613361870549302250 292730739620525741
194762025028783995 544210755574808776
556627093382336718 958921242265313514
546128508492571806 246866902946527408
393932605445808561 910938602210378539
914840793408902077 255591380129904024
195913568494820109 413922890673803940
619898150768195133 833342119974738160
87419457417016131 334850432627430109
288573793153649336 925610902626638070
522497537361063 813175909953633438
444026516979581152 954832972057956381
822716837190916298 147736366107774955
569653856760807902 184899487493930475
33776271666080414 478244567900174790
466492820071589565 737230754892342028
813693502197632807 857991042514859039
925055217040430421 777255773666660929
925390761464818405 144047739885020364
731001994063946005 705256424823360334
775543422405174762 439197536686505476
574908266045403041 182570338504305260
509500757818434866 827184399997892868
703622773028788890 475912833107766655
837707429744343452 248669168842571003
758438143566909292 529199674185184339
943866773510196534 395423406669250558
808158486059740979 473680965284413748
45955153741556993 123870699130069711
574986692757179971 903380700082664389
385669661847749062 72080604045137851
421954823717920161 752543363654290084
846136893458870269 37894276458005751
803707020089855759 695998151175201311
531676707900145850 789007983245646562
606047231471565654 70439312964486068
402883110573066585 51582406152454838
846885045244409991 77263051125058173
904523367200582330 805457623621917851
775249194986249497 325754501864595097
860052187218384017 882552932602401065
862956166184588101 196617106591357916
422786599592971683 589393967715205130
173629343289849030 153134298186557386
838058602812295645 911955570103774166
588321952719253248 340519672118978703
629776555338633351 346705215247861145
20692075115916321 362891870458791562
316582139581568783 492260177492890711
813342810209026779 899947428329635897
626360202983125153 550155990857425124
398024702813146825 860703287639447708
459756844573539700 430928878120763989
986596797590702462 234773395317001037
714850569342722029 798193916698026287
340020008680707541 312428455435886603
165099784952774959 802154011409398334
294086902988508998 152174905211325764
79690679684233140 713090969192886238
187060170571633096 401945782828607254
906610372080095947 17251522738628316
263346361666469745 882566671290491146
742292706568456827 705019841048254434
390010494545324648 170474535008451061
826890284258061021 465692499070022164
567261374931802093 462161615408794569
410417755509181940 839282585267337178
587967433570810902 150641208440217582
966409941953047438 680349315027906486
283760770098822373 24811235357874127
578594014381387703 379901893390085926
312876525235686809 922445746838851503
11008512074935660 38083669331985465
881124828506220666 111970093678348145
808593100664682923 519076215402375748
213261863399804744 595288711117368199
521870808896498598 655594400344937774
565802623488394969 676872041673744130
421302995234396729 726814378773365201
221508173241590393 635198646633144635
632341916555338446 45471874929059001
658876583038852268 602461324332081022
39632762797715829 378584287262677713
96517634649758994 755806481421679285
784954224056001695 351200279055944615
553838051632416105 494022337673243850
775348157782918686 263863565180980984
646867792666486352 105427297408401255
272874898956697235 89024096246497193
555483687463430574 141629213179923679
507547258968984633 606065774512597286
346220656420273427 346699023495848555
488543560598927864 292278619302267942
890219730531853737 685106393251255785
599370914138059624 595066845371134344
522172478015534667 740107606553721211
975739542093569198 810075083527374998
77609604540460678 445250581046338425
544297635542194986 990220864650422275
953055997203230078 24230666788783073
826690984151374088 464691310157658388
224467650322657114 562732669915226820
928294980970961061 245396633512720752
278718491141892270 197399022003059593
105587458253974502 36752006659944964
762982610382909493 628165509989514970
676994654686853704 381910823776661520
563436510834376641 128304837643614583
332466575868123709 423139507983818258
93925693785746878 346347534719513593
22389214382737731 596431348168053274
553522089724693615 814382131122069601
167942136334632443 64449610034933717
777542636701982482 491037730626469774
869440762562490826 994527525663402019
981939558416321276 35073878627634055
67687353001275260 158494671694154598
962072122445875771 716930377357078978
836759737311985141 565976546646647139
316079415703860829 342056854569467946
669205159984754175 593621626438851221
247076769622967295 237861372193287346
889147178616478001 49371499654083272
380719784535271082 612042823330808207
624996593619725903 292127633331443901
675929301784166553 467066397952827927
264109078995833681 552584044656399548
213013993583482695 922167889012058840
634713952501729843 585906952499250681
14451948782265138 292635550291726349
663582638905740791 752238926180093857
563078570436951676 711264432401972100
904077518098762298 185738097276529941
600590472379085628 752598112229925680
101426046767369511 913210775483454335
728050070362180048 834629815408327664
317985931977020770 638660480339101938
393374593925460968 781784635463492679
197960115439418194 138118745333884547
742064006914626757 776492792172962661
665269241670476551 568096879173801708
882255026213044227 769722312100423804
612138431456569949 531233257752991874
745542289676852214 732450425920617208
19341229395226919 3318783400940237
520416644890828108 803905456830738586
343034017422060129 946996587129521138
942879669061876369 445147361554245563
153835027121835812 777932854533175700
81987701577171577 184846188600087628
303036918445520853 698105496700976487
