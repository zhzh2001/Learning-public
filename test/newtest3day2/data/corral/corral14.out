315647599521698076
